clearwater-diags-monitor/usr/share/clearwater/bin/gather_diags_and_report_location

```diff
@@ -32,38 +32,88 @@ fi
 
 DIAGS_END_LOCATION=${CLEARWATER_USER_HOME}/ftp/dumps/
 
-if [ $# -ne 0 ]
+# Check for the force or force-confirmed options. These are used to run diags
+# gathering even if we have low idle CPU, or to do so and skip any confirmation.
+force=false
+confirmed=false
+if [ "$1" == "--force" ]
+then
+  force=true
+elif [ "$1" == "--force-confirmed" ]
+then
+  force=true
+  confirmed=true
+fi
+
+no_report=false
+# Also check to see if this is a call through from gather_diags
+if [[ "$1" == "--no-report" || "$2" == "--no-report" ]]
+then
+  no_report=true
+fi
```

Comment:
This argument parsing is quite nasty. Suggest we should be using `getopt` to parse these arguments - see http://manpages.ubuntu.com/manpages/trusty/man1/getopt.1.html